doc/TUTORIAL.md

```diff
@@ -170,8 +170,8 @@ This example is based on the [Ruby Gems version](https://github.com/badges/shiel
 ```js
 'use strict'                                                    // (1)
 
-const BaseJsonService = require('../base-json')                 // (2)
-const { renderVersionBadge } = require('../../lib/version')     // (3)
+const { renderVersionBadge } = require('../../lib/version')     // (2)
+const BaseJsonService = require('..')                           // (3)
```

Comment:
Should this be `const { BaseJsonService } = require('..')`?

Reply:
Gooood catch!